components/Layout/NextChakraLink.tsx

```diff
@@ -0,0 +1,14 @@
+import { PropsWithChildren } from 'react'
+import NextLink from 'next/link'
+import { LinkProps as NextLinkProps } from 'next/dist/client/link'
+import { Link as ChakraLink, LinkProps as ChakraLinkProps } from '@chakra-ui/core'
+
+export type NextChakraLinkProps = PropsWithChildren<NextLinkProps & Omit<ChakraLinkProps, 'as'>>
+
+export const NextChakraLink = ({ href, as, replace, scroll, shallow, prefetch, children, ...chakraProps }: NextChakraLinkProps) => {
```

Comment:
```suggestion
export function NextChakraLink({ href, as, replace, scroll, shallow, prefetch, children, ...chakraProps }: NextChakraLinkProps) {
```

Convert ke `function`.